src/libsystemd-network/network-internal.c

```diff
@@ -164,7 +170,7 @@ int config_parse_net_condition(const char *unit,
         return 0;
 }
 
-int config_parse_ifnames(
+int config_parse_match_strv(
```

Comment:
As I wrote in #12842, it'd be great to have a test for this, but it can be added in a later PR.